README.md

```diff
@@ -264,6 +264,14 @@ wfTracerBuilder.WithGlobalMultiValuedTags(new Dictionary<string, IEnumerable<str
 ITracer tracer = wfTracerBuilder.Build();
 ```
 
+#### Add Custom Span-Level RED metrics
+
+Optionally, you can add custom span-level tags to propagate RED metrics. See [Custom Span-Level Tags for RED Metrics](https://docs.wavefront.com/trace_data_details.html#custom-span-level-tags-for-red-metrics) for details.
+
+```csharp
+wfTracerBuilder.redMetricsCustomTagKeys("env", "location");
```

Comment:
should be `wfTracerBuilder.RedMetricsCustomTagKeys(new HashSet<string>{ "env", "location" });`

Reply:
Thank you! Updated.